x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/transactions/transactions_groups_alerts.spec.ts

```diff
@@ -208,6 +208,9 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
         });
 
         it('returns the correct number of alert counts', async () => {
+          // fails on MKI, see https://github.com/elastic/kibana/issues/201531
+          // @ts-expect-error
+          this.tags(['failsOnMKI']);
```

Comment:
These tags can only be assigned on suite level and not on test level. So this needs to move up to the `describe`. Also `this.tags` is only available in the `function` context, so you have to change it from something like
```
describe('xyz', () => {
```
to
```
describe('xyz', function () {
```

Reply:
fixed, thanks!